### ISSUE TITLE
RangeError: Maximum call stack size exceeded on Chrome 48

### ISSUE THREAD

**Issue by karls** [webrtcHacks/adapter#440]
Hey folks! Many thanks for this project! We received an error report through Rollbar this morning while one of our users was using our web app.

**Browsers and versions affected**
Google Chrome 48, on Windows. Using adapter.js version 3.1.3.

**Description**
One of our users was affected by an error that was later "fixed" by upgrading their browser. The error itself seems to originate from a call to `getUserMedia` and its error handler. We report any errors raised there to Rollbar (see attached screenshot).

**Steps to reproduce**
Difficult, I could not figure out how to downgrade to Chrome 48. Is it even possible?

**Expected results**
No error should have been reported and the user should have been able to continue what they were doing as this was a standard user flow through our app.

**Actual results**
User could not continue with their intended actions.

<img width="1220" alt="screen shot 2017-02-10 at 13 40 59" src="https://cloud.githubusercontent.com/assets/251402/22828776/11a201cc-ef97-11e6-83e3-68f25876d435.png">

**Comment by karls** [comment]
Another variation on the same error -- note the last two calls.

<img width="1204" alt="screen shot 2017-02-10 at 13 50 50" src="https://cloud.githubusercontent.com/assets/251402/22828983/0c208498-ef98-11e6-9407-484313ef431f.png">

**Comment by fippo** [comment]
that should be fixed with #420 -- can you check you deployed that? The line numbers seem to be off from the artifact that I get from latest master.
Are you using URL.createObjectURL?

I just tested in an old version of chrome 46 I found on https://commondatastorage.googleapis.com/chromium-browser-continuous/index.html?prefix=Linux_x64/339982/ and the patch seemed to work fine but couldn't find an 48 binary (48.0.2564.109 has a base position of 359700)

**Comment by karls** [comment]
Hi Philipp,

I should have checked that first! I was convinced that we were on 3.1.3, but we were, in fact, on 3.1.0. Very sorry! I have upgraded to 3.1.3 now.

I am using `URL.createObjectURL`, but that call would've happened further downstream. The error I reported originated from a call to `getUserMedia`, which happens before I call `URL.createObjectURL`.

I will keep an eye on this, but I suspect it's fixed now, given that I was using an earlier version of the library.

Many thanks!
